dev/docker-compose.yaml

```
@@ -86,6 +140,46 @@ services:
       - internal
 
 
+  logs:
+    image: postgrest/postgrest
+    environment:
+      PGRST_DB_URI: postgres://postgres:postgres@db:5432/app_db
+      PGRST_DB_SCHEMA: api
+      PGRST_DB_ANON_ROLE: web_anon
+    env_file:
+      - logs.env
+    depends_on:
+      - db
+    networks:
+      - internal
+
+
+  redis:
+    image: redis
+    networks:
+      - internal
+
+
+  screener:
+    image: ghcr.io/uwcirg/asbi-screening-app:${SCREENER_IMAGE_TAG:-latest}
+    env_file:
+      - screener.env
+    depends_on:
+      - fhir
+    labels:
+      - "traefik.enabled=true"
```

Traefik logs had a hint, but still was a bit of a puzzler:
```
 msg: Skip container asbi-safer-pbdev: field not found, node: enabled
```

```suggestion
      - "traefik.enable=true"
```

---

Thank you!